presto-main/src/main/java/com/facebook/presto/sql/analyzer/Analysis.java

```diff
@@ -96,12 +96,16 @@
 
     private Optional<Insert> insert = Optional.empty();
 
-    public Analysis(Statement root, List<Expression> parameters)
+    // for describe input and describe output
+    private final boolean isDescribe;
```

Comment:
it also gets used for describe output in the next PR, but `isDescribeInputOrOutput` seems too long.  Any ideas for a name that encompasses both?